Create Rule “software-engineers-should-only-roll-forward/rule” (#8872)
SSWConsulting/SSW.Rules.Content: rules/do-you-know-roll-forward-or-roll-backward/rule.md

---
type: rule
tips: ""
title: Do you know when to Roll Forward and when to Roll Backward?
seoDescription: Learn when to roll forward and when to roll backward in software deployment.

> uri: do-you-know-roll-forward-or-roll-backward

Reviewer: ```suggestion
uri: when-to-roll-forward-or-roll-backward
```
